### ISSUE TITLE
Remove all Luacheck warnings

### ISSUE THREAD

**Issue by lcpz** [lcpz/lain#475]
Following #474, it would be good to eliminate all warnings generated by [Luacheck](https://github.com/mpeterv/luacheck) (no ignored). As of 6b3bdca, they are 261:

```shell
Checking helpers.lua                              OK
Checking init.lua                                 OK
Checking layout/cascade.lua                       1 warning

    layout/cascade.lua:11:16: accessing undefined variable 'screen'

Checking layout/centerwork.lua                    8 warnings

    layout/centerwork.lua:13:71: accessing undefined variable 'mouse'
    layout/centerwork.lua:13:78: accessing undefined variable 'mousegrabber'
    layout/centerwork.lua:13:92: accessing undefined variable 'screen'
    layout/centerwork.lua:209:18: accessing undefined variable 'client'
    layout/centerwork.lua:211:24: accessing undefined variable 'client'
    layout/centerwork.lua:229:12: accessing undefined variable 'client'
    layout/centerwork.lua:229:29: accessing undefined variable 'client'
    layout/centerwork.lua:260:13: accessing undefined variable 'client'

Checking layout/init.lua                          OK
Checking layout/termfair.lua                      1 warning

    layout/termfair.lua:12:18: accessing undefined variable 'screen'

Checking util/dkjson.lua                          2 warnings

    util/dkjson.lua:61:7: unused variable '_ENV'
    util/dkjson.lua:324:21: setting non-standard global variable '_'

Checking util/init.lua                            2 warnings

    util/init.lua:17:22: accessing undefined variable 'client'
    util/init.lua:119:121: line is too long (123 > 120)

Checking util/markup.lua                          OK
Checking util/menu_iterator.lua                   3 warnings

    util/menu_iterator.lua:62:16: setting non-standard global variable '_'
    util/menu_iterator.lua:69:23: accessing undefined variable 'mouse'
    util/menu_iterator.lua:104:121: line is too long (130 > 120)

Checking util/quake.lua                           2 warnings

    util/quake.lua:10:33: accessing undefined variable 'client'
    util/quake.lua:14:22: accessing undefined variable 'screen'

Checking util/separators.lua                      OK
Checking widget/alsa.lua                          3 warnings

    widget/alsa.lua:40:17: setting non-standard global variable 'volume_now'
    widget/alsa.lua:41:17: setting non-standard global variable 'widget'
    widget/alsa.lua:43:29: accessing undefined variable 'volume_now'

Checking widget/alsabar.lua                       1 warning

    widget/alsabar.lua:98:17: setting non-standard global variable 'volume_now'

Checking widget/bat.lua                           41 warnings

    widget/bat.lua:53:5: setting non-standard global variable 'bat_notification_critical_preset'
    widget/bat.lua:61:5: setting non-standard global variable 'bat_notification_low_preset'
    widget/bat.lua:69:5: setting non-standard global variable 'bat_notification_charged_preset'
    widget/bat.lua:77:5: setting non-standard global variable 'bat_now'
    widget/bat.lua:85:5: mutating non-standard global variable 'bat_now'
    widget/bat.lua:86:5: mutating non-standard global variable 'bat_now'
    widget/bat.lua:88:9: mutating non-standard global variable 'bat_now'
    widget/bat.lua:89:9: mutating non-standard global variable 'bat_now'
    widget/bat.lua:124:17: mutating non-standard global variable 'bat_now'
    widget/bat.lua:125:17: mutating non-standard global variable 'bat_now'
    widget/bat.lua:125:60: accessing undefined variable 'bat_now'
    widget/bat.lua:140:9: mutating non-standard global variable 'bat_now'
    widget/bat.lua:140:26: accessing undefined variable 'bat_now'
    widget/bat.lua:141:32: accessing undefined variable 'bat_now'
    widget/bat.lua:143:17: mutating non-standard global variable 'bat_now'
    widget/bat.lua:146:9: mutating non-standard global variable 'bat_now'
    widget/bat.lua:148:12: accessing undefined variable 'bat_now'
    widget/bat.lua:149:16: accessing undefined variable 'bat_now'
    widget/bat.lua:149:69: accessing undefined variable 'bat_now'
    widget/bat.lua:150:17: mutating non-standard global variable 'bat_now'
    widget/bat.lua:151:17: mutating non-standard global variable 'bat_now'
    widget/bat.lua:152:17: mutating non-standard global variable 'bat_now'
    widget/bat.lua:155:20: accessing undefined variable 'bat_now'
    widget/bat.lua:161:24: accessing undefined variable 'bat_now'
    widget/bat.lua:168:25: setting non-standard global variable 'rate_time_magnitude'
    widget/bat.lua:169:53: accessing undefined variable 'rate_time_magnitude'
    widget/bat.lua:175:17: mutating non-standard global variable 'bat_now'
    widget/bat.lua:176:17: mutating non-standard global variable 'bat_now'
    widget/bat.lua:177:17: mutating non-standard global variable 'bat_now'
    widget/bat.lua:178:20: accessing undefined variable 'bat_now'
    widget/bat.lua:179:17: mutating non-standard global variable 'bat_now'
    widget/bat.lua:180:17: mutating non-standard global variable 'bat_now'
    widget/bat.lua:181:17: mutating non-standard global variable 'bat_now'
    widget/bat.lua:185:9: setting non-standard global variable 'widget'
    widget/bat.lua:190:16: accessing undefined variable 'bat_now'
    widget/bat.lua:191:29: accessing undefined variable 'bat_now'
    widget/bat.lua:193:34: accessing undefined variable 'bat_notification_critical_preset'
    widget/bat.lua:196:33: accessing undefined variable 'bat_now'
    widget/bat.lua:198:34: accessing undefined variable 'bat_notification_low_preset'
    widget/bat.lua:203:20: accessing undefined variable 'bat_now'
    widget/bat.lua:205:30: accessing undefined variable 'bat_notification_charged_preset'

Checking widget/cal.lua                           6 warnings

    widget/cal.lua:54:121: line is too long (137 > 120)
    widget/cal.lua:55:121: line is too long (141 > 120)
    widget/cal.lua:56:121: line is too long (124 > 120)
    widget/cal.lua:65:121: line is too long (121 > 120)
    widget/cal.lua:77:121: line is too long (128 > 120)
    widget/cal.lua:94:121: line is too long (157 > 120)

Checking widget/contrib/init.lua                  OK
Checking widget/contrib/moc.lua                   23 warnings

    widget/contrib/moc.lua:32:5: setting non-standard global variable 'moc_notification_preset'
    widget/contrib/moc.lua:38:13: setting non-standard global variable 'moc_now'
    widget/contrib/moc.lua:50:52: mutating non-standard global variable 'moc_now'
    widget/contrib/moc.lua:51:52: mutating non-standard global variable 'moc_now'
    widget/contrib/moc.lua:52:52: mutating non-standard global variable 'moc_now'
    widget/contrib/moc.lua:53:52: mutating non-standard global variable 'moc_now'
    widget/contrib/moc.lua:54:52: mutating non-standard global variable 'moc_now'
    widget/contrib/moc.lua:55:52: mutating non-standard global variable 'moc_now'
    widget/contrib/moc.lua:56:52: mutating non-standard global variable 'moc_now'
    widget/contrib/moc.lua:61:13: mutating non-standard global variable 'moc_notification_preset'
    widget/contrib/moc.lua:61:78: accessing undefined variable 'moc_now'
    widget/contrib/moc.lua:62:44: accessing undefined variable 'moc_now'
    widget/contrib/moc.lua:62:59: accessing undefined variable 'moc_now'
    widget/contrib/moc.lua:62:74: accessing undefined variable 'moc_now'
    widget/contrib/moc.lua:63:13: setting non-standard global variable 'widget'
    widget/contrib/moc.lua:66:16: accessing undefined variable 'moc_now'
    widget/contrib/moc.lua:67:20: accessing undefined variable 'moc_now'
    widget/contrib/moc.lua:68:58: accessing undefined variable 'moc_now'
    widget/contrib/moc.lua:70:39: mutating non-standard global variable 'moc_notification_preset'
    widget/contrib/moc.lua:73:39: accessing undefined variable 'moc_notification_preset'
    widget/contrib/moc.lua:79:83: accessing undefined variable 'moc_now'
    widget/contrib/moc.lua:80:121: line is too long (122 > 120)
    widget/contrib/moc.lua:86:21: accessing undefined variable 'moc_now'

Checking widget/contrib/redshift.lua              OK
Checking widget/contrib/task.lua                  1 warning

    widget/contrib/task.lua:12:17: accessing undefined variable 'mouse'

Checking widget/contrib/tp_smapi.lua              14 warnings

    widget/contrib/tp_smapi.lua:44:121: line is too long (154 > 120)
    widget/contrib/tp_smapi.lua:112:9: setting non-standard global variable 'tpbat'
    widget/contrib/tp_smapi.lua:117:18: mutating non-standard global variable 'tpbat'
    widget/contrib/tp_smapi.lua:118:13: setting non-standard global variable 'tpbat_now'
    widget/contrib/tp_smapi.lua:126:17: mutating non-standard global variable 'tpbat_now'
    widget/contrib/tp_smapi.lua:127:17: mutating non-standard global variable 'tpbat_now'
    widget/contrib/tp_smapi.lua:128:17: mutating non-standard global variable 'tpbat_now'
    widget/contrib/tp_smapi.lua:130:24: accessing undefined variable 'tpbat_now'
    widget/contrib/tp_smapi.lua:131:21: mutating non-standard global variable 'tpbat_now'
    widget/contrib/tp_smapi.lua:131:40: accessing undefined variable 'tpbat_now'
    widget/contrib/tp_smapi.lua:135:13: setting non-standard global variable 'widget'
    widget/contrib/tp_smapi.lua:135:22: accessing undefined variable 'tpbat'
    widget/contrib/tp_smapi.lua:139:57: accessing undefined variable 'tpbat'
    widget/contrib/tp_smapi.lua:141:16: accessing undefined variable 'tpbat'

Checking widget/cpu.lua                           4 warnings

    widget/cpu.lua:63:9: setting non-standard global variable 'cpu_now'
    widget/cpu.lua:64:9: mutating non-standard global variable 'cpu_now'
    widget/cpu.lua:64:25: accessing undefined variable 'cpu_now'
    widget/cpu.lua:65:9: setting non-standard global variable 'widget'

Checking widget/fs.lua                            13 warnings

    widget/fs.lua:76:9: setting non-standard global variable 'fs_now'
    widget/fs.lua:92:21: mutating non-standard global variable 'fs_now'
    widget/fs.lua:100:24: accessing undefined variable 'fs_now'
    widget/fs.lua:111:9: setting non-standard global variable 'widget'
    widget/fs.lua:114:26: accessing undefined variable 'fs_now'
    widget/fs.lua:114:48: accessing undefined variable 'fs_now'
    widget/fs.lua:119:93: accessing undefined variable 'fs_now'
    widget/fs.lua:119:121: line is too long (121 > 120)
    widget/fs.lua:131:68: accessing undefined variable 'fs_now'
    widget/fs.lua:131:93: accessing undefined variable 'fs_now'
    widget/fs.lua:131:112: accessing undefined variable 'fs_now'
    widget/fs.lua:131:121: line is too long (149 > 120)
    widget/fs.lua:131:131: accessing undefined variable 'fs_now'

Checking widget/imap.lua                          15 warnings

    widget/imap.lua:39:5: setting non-standard global variable 'mail_notification_preset'
    widget/imap.lua:68:13: setting non-standard global variable 'imap_now'
    widget/imap.lua:70:53: mutating non-standard global variable 'imap_now'
    widget/imap.lua:71:13: setting non-standard global variable 'mailcount'
    widget/imap.lua:71:25: accessing undefined variable 'imap_now'
    widget/imap.lua:72:13: setting non-standard global variable 'widget'
    widget/imap.lua:76:35: accessing undefined variable 'mailcount'
    widget/imap.lua:76:49: accessing undefined variable 'mailcount'
    widget/imap.lua:76:68: accessing undefined variable 'mailcount'
    widget/imap.lua:77:35: mutating non-standard global variable 'mail_notification_preset'
    widget/imap.lua:79:30: accessing undefined variable 'mail_notification_preset'
    widget/imap.lua:80:84: accessing undefined variable 'mailcount'
    widget/imap.lua:80:95: accessing undefined variable 'mailcount'
    widget/imap.lua:80:121: line is too long (123 > 120)
    widget/imap.lua:84:35: accessing undefined variable 'imap_now'

Checking widget/init.lua                          OK
Checking widget/mem.lua                           21 warnings

    widget/mem.lua:24:9: setting non-standard global variable 'mem_now'
    widget/mem.lua:27:49: mutating non-standard global variable 'mem_now'
    widget/mem.lua:28:49: mutating non-standard global variable 'mem_now'
    widget/mem.lua:29:49: mutating non-standard global variable 'mem_now'
    widget/mem.lua:30:49: mutating non-standard global variable 'mem_now'
    widget/mem.lua:31:49: mutating non-standard global variable 'mem_now'
    widget/mem.lua:32:49: mutating non-standard global variable 'mem_now'
    widget/mem.lua:33:49: mutating non-standard global variable 'mem_now'
    widget/mem.lua:38:9: mutating non-standard global variable 'mem_now'
    widget/mem.lua:38:24: accessing undefined variable 'mem_now'
    widget/mem.lua:38:40: accessing undefined variable 'mem_now'
    widget/mem.lua:38:55: accessing undefined variable 'mem_now'
    widget/mem.lua:38:69: accessing undefined variable 'mem_now'
    widget/mem.lua:38:85: accessing undefined variable 'mem_now'
    widget/mem.lua:39:9: mutating non-standard global variable 'mem_now'
    widget/mem.lua:39:28: accessing undefined variable 'mem_now'
    widget/mem.lua:39:43: accessing undefined variable 'mem_now'
    widget/mem.lua:40:9: mutating non-standard global variable 'mem_now'
    widget/mem.lua:40:35: accessing undefined variable 'mem_now'
    widget/mem.lua:40:50: accessing undefined variable 'mem_now'
    widget/mem.lua:42:9: setting non-standard global variable 'widget'

Checking widget/mpd.lua                           34 warnings

    widget/mpd.lua:26:121: line is too long (121 > 120)
    widget/mpd.lua:41:5: setting non-standard global variable 'mpd_notification_preset'
    widget/mpd.lua:47:13: setting non-standard global variable 'mpd_now'
    widget/mpd.lua:69:55: mutating non-standard global variable 'mpd_now'
    widget/mpd.lua:70:55: mutating non-standard global variable 'mpd_now'
    widget/mpd.lua:71:55: mutating non-standard global variable 'mpd_now'
    widget/mpd.lua:72:55: mutating non-standard global variable 'mpd_now'
    widget/mpd.lua:73:55: mutating non-standard global variable 'mpd_now'
    widget/mpd.lua:74:55: mutating non-standard global variable 'mpd_now'
    widget/mpd.lua:75:55: mutating non-standard global variable 'mpd_now'
    widget/mpd.lua:76:55: mutating non-standard global variable 'mpd_now'
    widget/mpd.lua:77:55: mutating non-standard global variable 'mpd_now'
    widget/mpd.lua:78:55: mutating non-standard global variable 'mpd_now'
    widget/mpd.lua:79:55: mutating non-standard global variable 'mpd_now'
    widget/mpd.lua:80:55: mutating non-standard global variable 'mpd_now'
    widget/mpd.lua:81:55: mutating non-standard global variable 'mpd_now'
    widget/mpd.lua:82:55: mutating non-standard global variable 'mpd_now'
    widget/mpd.lua:83:55: mutating non-standard global variable 'mpd_now'
    widget/mpd.lua:84:55: mutating non-standard global variable 'mpd_now'
    widget/mpd.lua:85:55: mutating non-standard global variable 'mpd_now'
    widget/mpd.lua:90:13: mutating non-standard global variable 'mpd_notification_preset'
    widget/mpd.lua:90:78: accessing undefined variable 'mpd_now'
    widget/mpd.lua:91:44: accessing undefined variable 'mpd_now'
    widget/mpd.lua:91:59: accessing undefined variable 'mpd_now'
    widget/mpd.lua:91:73: accessing undefined variable 'mpd_now'
    widget/mpd.lua:92:13: setting non-standard global variable 'widget'
    widget/mpd.lua:95:16: accessing undefined variable 'mpd_now'
    widget/mpd.lua:96:39: accessing undefined variable 'mpd_now'
    widget/mpd.lua:97:58: accessing undefined variable 'mpd_now'
    widget/mpd.lua:99:39: mutating non-standard global variable 'mpd_notification_preset'
    widget/mpd.lua:102:39: accessing undefined variable 'mpd_notification_preset'
    widget/mpd.lua:108:41: accessing undefined variable 'mpd_now'
    widget/mpd.lua:109:87: accessing undefined variable 'mpd_now'
    widget/mpd.lua:122:20: accessing undefined variable 'mpd_now'

Checking widget/net.lua                           18 warnings

    widget/net.lua:44:9: setting non-standard global variable 'net_now'
    widget/net.lua:54:121: line is too long (125 > 120)
    widget/net.lua:55:121: line is too long (125 > 120)
    widget/net.lua:63:13: mutating non-standard global variable 'net_now'
    widget/net.lua:63:32: accessing undefined variable 'net_now'
    widget/net.lua:64:13: mutating non-standard global variable 'net_now'
    widget/net.lua:64:32: accessing undefined variable 'net_now'
    widget/net.lua:72:121: line is too long (127 > 120)
    widget/net.lua:75:121: line is too long (128 > 120)
    widget/net.lua:81:121: line is too long (126 > 120)
    widget/net.lua:102:13: mutating non-standard global variable 'net_now'
    widget/net.lua:103:13: mutating non-standard global variable 'net_now'
    widget/net.lua:104:13: mutating non-standard global variable 'net_now'
    widget/net.lua:109:9: mutating non-standard global variable 'net_now'
    widget/net.lua:109:46: accessing undefined variable 'net_now'
    widget/net.lua:110:9: mutating non-standard global variable 'net_now'
    widget/net.lua:110:50: accessing undefined variable 'net_now'
    widget/net.lua:112:9: setting non-standard global variable 'widget'

Checking widget/pulse.lua                         10 warnings

    widget/pulse.lua:25:121: line is too long (179 > 120)
    widget/pulse.lua:30:13: setting non-standard global variable 'volume_now'
    widget/pulse.lua:36:28: accessing undefined variable 'volume_now'
    widget/pulse.lua:39:13: mutating non-standard global variable 'volume_now'
    widget/pulse.lua:41:17: mutating non-standard global variable 'volume_now'
    widget/pulse.lua:45:13: mutating non-standard global variable 'volume_now'
    widget/pulse.lua:45:32: accessing undefined variable 'volume_now'
    widget/pulse.lua:46:13: mutating non-standard global variable 'volume_now'
    widget/pulse.lua:46:32: accessing undefined variable 'volume_now'
    widget/pulse.lua:48:13: setting non-standard global variable 'widget'

Checking widget/pulsebar.lua                      11 warnings

    widget/pulsebar.lua:54:121: line is too long (201 > 120)
    widget/pulsebar.lua:79:13: setting non-standard global variable 'volume_now'
    widget/pulsebar.lua:85:31: accessing undefined variable 'volume_now'
    widget/pulsebar.lua:88:13: mutating non-standard global variable 'volume_now'
    widget/pulsebar.lua:90:15: mutating non-standard global variable 'volume_now'
    widget/pulsebar.lua:94:13: mutating non-standard global variable 'volume_now'
    widget/pulsebar.lua:94:32: accessing undefined variable 'volume_now'
    widget/pulsebar.lua:95:13: mutating non-standard global variable 'volume_now'
    widget/pulsebar.lua:95:32: accessing undefined variable 'volume_now'
    widget/pulsebar.lua:97:26: accessing undefined variable 'volume_now'
    widget/pulsebar.lua:98:26: accessing undefined variable 'volume_now'

Checking widget/sysload.lua                       4 warnings

    widget/sysload.lua:28:9: setting non-standard global variable 'load_1'
    widget/sysload.lua:28:17: setting non-standard global variable 'load_5'
    widget/sysload.lua:28:25: setting non-standard global variable 'load_15'
    widget/sysload.lua:30:9: setting non-standard global variable 'widget'

Checking widget/temp.lua                          5 warnings

    widget/temp.lua:25:13: setting non-standard global variable 'temp_now'
    widget/temp.lua:31:21: mutating non-standard global variable 'temp_now'
    widget/temp.lua:34:13: setting non-standard global variable 'coretemp_now'
    widget/temp.lua:34:28: accessing undefined variable 'temp_now'
    widget/temp.lua:35:13: setting non-standard global variable 'widget'

Checking widget/weather.lua                       18 warnings

    widget/weather.lua:29:121: line is too long (147 > 120)
    widget/weather.lua:30:121: line is too long (161 > 120)
    widget/weather.lua:94:13: setting non-standard global variable 'weather_now'
    widget/weather.lua:94:26: setting non-standard global variable '_'
    widget/weather.lua:96:33: accessing undefined variable 'weather_now'
    widget/weather.lua:96:70: accessing undefined variable 'weather_now'
    widget/weather.lua:98:28: accessing undefined variable 'weather_now'
    widget/weather.lua:100:71: accessing undefined variable 'weather_now'
    widget/weather.lua:101:28: accessing undefined variable 'weather_now'
    widget/weather.lua:113:13: setting non-standard global variable 'weather_now'
    widget/weather.lua:113:26: setting non-standard global variable '_'
    widget/weather.lua:115:33: accessing undefined variable 'weather_now'
    widget/weather.lua:115:70: accessing undefined variable 'weather_now'
    widget/weather.lua:116:42: accessing undefined variable 'weather_now'
    widget/weather.lua:117:42: accessing undefined variable 'weather_now'
    widget/weather.lua:118:33: accessing undefined variable 'weather_now'
    widget/weather.lua:128:17: setting non-standard global variable 'widget'
    widget/weather.lua:142:121: line is too long (124 > 120)

Total: 261 warnings / 0 errors in 33 files
```

**Comment by staticssleever668** [comment]
To fix some of them [.luacheckrc from Awesome repository](https://github.com/awesomeWM/awesome/blob/master/.luacheckrc) is needed

**Comment by lcpz** [comment]
Closed by 1703661.
